### PR TITLE
Add model names to WIT model comparison mode

### PR DESCRIPTION
**Pull request by jameswex**
* Motivation for features / changes

Use model names in display when comparing two models in WIT, instead of model numbers, for clarity.

* Technical description of changes

Model names from the setup UI are now used for display purposes instead of model index. If model names are identical, adds index to the name for clarity.

In notebook mode, model names can be set through the existing set_model_name/set_compare_model_name methods, and now those names are used in the display.

* Screenshots of UI changes

Example UI change showing model names in display:
modelnames

* Detailed steps to verify changes work correctly (as executed by you)

Ran single model demos and confirmed no visual changes. 
Ran multiple model demo and confirmed appropriate model names appeared in all appropriate parts of display (facets dive dropdows, inference viewer, PD plots, performance results).
Built pip package and verified in example jupyter notebooks as well.

* Alternate designs / implementations considered

### CODE REVIEW

**Comment by jameswex** [comment]
@tolga-b Please review

File: tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html [hunk 1]
```diff
@@ -1947,6 +1951,18 @@ <h2>Create a distance feature</h2>
         }
       },
 
+      modelNameChanged_: function() {
```
[the remark this answers is not in the document]
Comment (jameswex) [reply tensorboard/plugins/interactive_inference/tf_interactive_inference_dashboard/tf-interactive-inference-dashboard.html]: fixed this

**Comment by jameswex** [comment]
fixed splitChartKey and modelNameChanged_

**Comment by jameswex** [comment]
@wchargin for review as you are on-call. tolga has approved from the WIT-side.

**Comment by wchargin** [comment]
Looks fine to me; feel free to merge at your convenience.

**Pull request merged**